python/ray/rllib/optimizers/async_samples_optimizer.py

```diff
@@ -60,6 +62,7 @@ def __init__(self,
                     "parallel data loader buffers as minibatch buffers: "
                     "{} vs {}".format(num_data_loader_buffers,
                                       minibatch_buffer_size))
+
```

Comment:
@michaelzhiluo

Reply:
format.sh

Reply:
everything in this file

Reply:
format.sh wont randomly introduce a space especially if this file doesnt actually need to be in the PR diff